tests/unit/dojson/test_dojson_experiments.py

```diff
@@ -29,210 +30,212 @@
 
 
-def test_contact_details_from_multiple_marcxml_270():
+def test_experiment_names_from_marcxml_119__c():
     schema = load_schema('experiments')
-    subschema = schema['properties']['contact_details']
+    subschema = schema['properties']['experiment']
 
     snippet = (
-        '<record> '
-        '  <datafield tag="270" ind1=" " ind2=" ">'
-        '    <subfield code="m">lindner@mpi-hd.mpg.de</subfield>'
-        '    <subfield code="p">Manfred Lindner</subfield>'
-        '  </datafield>'
-        '  <datafield tag="270" ind1=" " ind2=" ">'
-        '    <subfield code="p">Wynton Marsalis</subfield>'
+        '<record>'
+        '  <datafield tag="119" ind1=" " ind2=" ">'
+        '    <subfield code="a">CERN-ALPHA</subfield>'
         '  </datafield>'
         '</record>'
     )
 
-    expected = [
-        {
-            'name': 'Manfred Lindner',
-            'email': 'lindner@mpi-hd.mpg.de',
-        },
-        {
-            'name': 'Wynton Marsalis',
-        },
-    ]
+    expected = {
+            'legacy_name': 'CERN-ALPHA',
+            'value': 'ALPHA',
+        }
+
     result = experiments.do(create_record(snippet))
 
-    assert validate(result['contact_details'], subschema) is None
-    assert expected == result['contact_details']
+    assert validate(result['experiment'], subschema) is None
+    assert expected == result['experiment']
 
 
-def test_experiment_names_from_marcxml_119():
+def test_experiment_names_for_project_type_from_marcxml_119():
     schema = load_schema('experiments')
-    subschema = schema['properties']['experiment_names']
+    subschema = schema['properties']['project_type']
 
     snippet = (
         '<record>'
         '  <datafield tag="119" ind1=" " ind2=" ">'
         '    <subfield code="a">CERN-ALPHA</subfield>'
+        '    <subfield code="c">NA61</subfield>'
         '  </datafield>'
         '</record>'
     )
 
-    expected = [
-        {
-            'title': 'CERN-ALPHA',
-        },
-    ]
+    expected = ['experiment']
+
     result = experiments.do(create_record(snippet))
 
-    assert validate(result['experiment_names'], subschema) is None
-    assert expected == result['experiment_names']
+    assert validate(result['project_type'], subschema) is None
+    assert expected == result['project_type']
 
 
 def test_experiment_names_and_affiliations_from_marcxml_119():
     schema = load_schema('experiments')
-    subschema = schema['properties']['experiment_names']
+    subschema = schema['properties']['experiment']
+    subschema_acc = schema['properties']['accelerator']
+    subschema_inst = schema['properties']['institution']
 
     snippet = (
         '<record>'
         '  <datafield tag="119" ind1=" " ind2=" ">'
         '    <subfield code="a">CERN-ALPHA</subfield>'
         '    <subfield code="u">CERN</subfield>'
-        '    <subfield code="z">902725</subfield>'
+        '    <subfield code="c">NA61</subfield>'
+        '    <subfield code="b">LHC</subfield>'
```

Comment:
Never modify previous tests: they are there to guarantee that you're not breaking earlier behavior.